docker/fuzzbench/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
```

Comment:
We usually avoid directories with name of project. project name is not informative as a directory. Is this for the main server running, distributing tasks ? then server or something else sounds better.

Reply:
We call it fuzzbench because the image is shared between all the different "fuzzbench containers". Currently these are the main `run_experiment` container and the `worker` container (see the compose/fuzzbench.yaml file above how both containers use the same image). Later we might add e.g. a separate `report_generator` container to separate the report generation functionality during the experiment, which would also use the same image. So basically it's a "generic" image with a python runtime, our python deps, and the fuzzbench codebase. Does that answer your question?

Reply:
Yes generic image is better named as base (and not project name), but i guess there is already a base-image directory. I am fine to keep it as-is for now, we can rename it later when other code is removed.